project-base/tests/FrontendApiBundle/Functional/Article/GetArticleTest.php

```diff
@@ -67,6 +102,39 @@ private function getArticleDataProvider(): array
         return $data;
     }
 
+    /**
+     * @return array
+     */
+    public function getSpecialArticleDataProvider(): array
+    {
+        return [
+            [
+                'termsAndConditionsArticle',
+                [
+                    'name' => 'Terms and conditions',
+                    'placement' => Article::PLACEMENT_FOOTER,
+                    'text' => 'Morbi posuere mauris dolor, quis accumsan dolor ullamcorper eget. Phasellus at elementum magna, et pretium neque. Praesent tristique lorem mi, eget varius quam aliquam eget. Vivamus ultrices interdum nisi, sed placerat lectus fermentum non. Phasellus ac quam vitae nisi aliquam vestibulum. Sed rhoncus tortor a arcu sagittis placerat. Nulla lectus nunc, ultrices ac faucibus sed, accumsan nec diam. Nam auctor neque quis tincidunt tempus. Nunc eget risus tristique, lobortis metus vitae, pellentesque leo. Vivamus placerat turpis ac dolor vehicula tincidunt. Sed venenatis, ante id ultrices convallis, lacus elit porttitor dolor, non porta risus ipsum ac justo. Integer id pretium quam, id placerat nulla.',
```

Comment:
Why are you providing only some data when you are querying all data in getSpecialArticleQuery method?

Reply:
fixed